ui/webui/test/check-basic

```diff
@@ -40,6 +41,11 @@ class TestBasic(MachineCase):
             ["False", "false", "True", "true"]
         )
 
+        # Check betanag
+        value = m.execute("cat /.buildstamp | grep IsFinal=")
+        if value.split("=", 1)[1] in ("False", "false"):
+            b.wait_visible("#betanag-icon")
+
```

Comment:
I would add an else as well.
```
else:
  b.wait_not_present("#betang-icon")
```

So that you make sure this actually covers all scenarios.
But it can be considered follow up material.